rebar.config

```diff
@@ -18,7 +18,7 @@
   {exml, ".*", {git, "git://github.com/esl/exml.git", "2.2.0"}},
   {lager, "2.0.3", {git, "git://github.com/basho/lager.git", "b7984d4"}},
   {cowboy, ".*", {git, "git://github.com/ninenines/cowboy.git", "1.0.4"}},
-  {exometer, ".*", {git, "git://github.com/esl/exometer.git", {branch, "1.1-patched"}}},
+  {exometer, ".*", {git, "git://github.com/esl/exometer.git", {tag, "1.2"}}},
```

Comment:
Let's use the main exometer repo which is Feuerlabs/exometer. And the newest version there is 1.2.1